### ISSUE TITLE
[NicoNico] 403 Will not download link, plays perfectly in browser.

### ISSUE THREAD

**Issue by Anon790** [ytdl-org/youtube-dl#28470]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2021.03.14**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '-u', 'PRIVATE', '-p', 'PRIVATE', 'https://www.nicovideo.jp/watch/sm36505961']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.03.14
[debug] Python version 3.9.2 (CPython) - macOS-10.14.6-x86_64-i386-64bit
[debug] exe versions: ffmpeg 4.3.2, ffprobe 4.3.2, rtmpdump 2.4
[debug] Proxy map: {}
[niconico] Logging in
[niconico] sm36505961: Downloading webpage
[niconico] sm36505961: Downloading flv info
[niconico] sm36505961: Downloading video info page
[debug] Default format spec: bestvideo+bestaudio/best
[debug] Invoking downloader on 'http://smile-cow63.nicovideo.jp/smile?m=36505961.29368'
ERROR: unable to download video data: HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "/usr/local/Cellar/youtube-dl/2021.3.14/libexec/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 1967, in process_info
    success = dl(filename, info_dict)
  File "/usr/local/Cellar/youtube-dl/2021.3.14/libexec/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 1906, in dl
    return fd.download(name, info)
  File "/usr/local/Cellar/youtube-dl/2021.3.14/libexec/lib/python3.9/site-packages/youtube_dl/downloader/common.py", line 366, in download
    return self.real_download(filename, info_dict)
  File "/usr/local/Cellar/youtube-dl/2021.3.14/libexec/lib/python3.9/site-packages/youtube_dl/downloader/http.py", line 351, in real_download
    establish_connection()
  File "/usr/local/Cellar/youtube-dl/2021.3.14/libexec/lib/python3.9/site-packages/youtube_dl/downloader/http.py", line 116, in establish_connection
    raise err
  File "/usr/local/Cellar/youtube-dl/2021.3.14/libexec/lib/python3.9/site-packages/youtube_dl/downloader/http.py", line 110, in establish_connection
    ctx.data = self.ydl.urlopen(request)
  File "/usr/local/Cellar/youtube-dl/2021.3.14/libexec/lib/python3.9/site-packages/youtube_dl/YoutubeDL.py", line 2279, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/local/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 523, in open
    response = meth(req, response)
  File "/usr/local/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 632, in http_response
    response = self.parent.error(
  File "/usr/local/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 561, in error
    return self._call_chain(*args)
  File "/usr/local/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 494, in _call_chain
    result = func(*args)
  File "/usr/local/Cellar/python@3.9/3.9.2_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 641, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

## Description

Basically I went into terminal on mac to download a video file off niconico, I did my normal command line prompt: youtube-dl then the niconico link which in this case it would've been: https://www.nicovideo.jp/watch/sm36505961 and it was giving me an error saying that niconico videos require login now, so I used my login with the extra strings or however it's called. -u -p and then added the link and it gave me a 403 forbidden error so now here I am. 
To create an account on niconico you can sign in with your google account or you can easily sign up with your email and password, there are other options to available on the website.

**Comment by october262** [comment]
while logged in, retry the link the cookies command.

**Comment by Anon790** [comment]
> while logged in, retry the link the cookies command.

Same error.

**Comment by animelover1984** [comment]
Related: https://github.com/animelover1984/youtube-dl/issues/29

**Comment by Anon790** [comment]
> Related: https://github.com/animelover1984/youtube-dl/issues/29

oh Thanks!
